src/css/styles.css

```diff
@@ -1,3 +1,106 @@
+#title {
+    text-align: center;
+    font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
+    font-weight: bold;
+    font-size: 40px;
+}
+
+#timelineButton {
+    align-items: left;
+    background-color: white;
+    border-radius: 12px;
+    border: 1px solid;
+    color: black;
+    font-size: 15px;
+    -webkit-transition-duration: 0.4s;
+    transition-duration: 0.4s;
+    justify-content: center;
+    padding: 5px 10px;
+}
+
+#timelineButton:hover {
+    background-color: #008CBA;
+    color: white;
+}
+
+#data {
+    text-align: center;
+    border-style: solid;
+    border-width: 1.5px;
+    border-color: black;
+    width: 80%;
+    margin: auto;
+    font-style: "Arial";
+    font-size: 25px;
+    font-weight: bold;
+}
+
+#data:empty {
+    display: none;
+}
+
+#errorMessage {
+    color: red;
+}
+
 .item:nth-child(even) {
-    background-color: grey;
+    background-color: #e9e9e9;
+}
+
+.item {
+    background-color: #e8f5fd;
+    align-items: center;
+    padding: 2px;
+    overflow: hidden;
+    display: flex;
+    border-style: solid;
+    border-width: .5px;
+    border-color: #C0C0C0;
+    border-top: 0;
+    border-left: 0;
+    border-right: 0;
+}
+
+.user {
+    float: left;
+    flex: 0 0 17%;
+}
+
+.twitterHandle {
+    font-size: 20px;
+    font-weight: bold;
+}
+
+.twitterName {
+    font-style: italic;
+    padding-bottom: 5px;
+}
+
+.message {
+    text-align: left;
+}
+
+.date {
+    font-style: italic;
+    font-family: "Arial";
+    font-size: 10pt;
+    padding-bottom: 5px;
+}
+
+.messageText {
+    font-size: 17px;
+}
+
+.buttonContainer {
+    text-align: right;
+    padding: 5px;
+    padding-right: 10%;
```

Comment:
why percentage as opposed to px?

Reply:
So it stays lined up with the output box if you resize the window. If it's set to a default px, it will gradually move left (in relation to the box) as the window gets narrower